test/condition-travis-test.js

```diff
@@ -98,7 +98,7 @@ test.serial('travis-deploy-once resolves with true', async t => {
 
   t.falsy(result);
   t.true(travisDeployOnce.calledOnce);
-  t.deepEqual(travisDeployOnce.firstCall.args[0], {travisOpts: {pro}});
+  t.deepEqual(travisDeployOnce.firstCall.args[0], {travisOpts: {pro, enterprise: undefined}});
```

Comment:
There is no need for this change. If `enterprise` is not defined it's the same as `undefined`

Reply:
no, the test failed without adding it

Reply:
Yes indeed. I though `deepEqual` would be smarter.